src/EA.Weee.DataAccess.Tests.Integration/NotesDataAccessIntegration.cs

```diff
@@ -571,7 +574,123 @@ public async Task GetNoteCountByStatusAndAatf_GivenStatusAndAatf_ShouldReturnCor
             }
         }
 
-        private async Task<Note> SetupSingleNote(WeeeContext context, DatabaseWrapper database, NoteType noteType = null)
+        [Fact]
+        public async Task GetNotesToTransfer_GivenSchemeAndCategories_NotesShouldBeReturned()
```

Comment:
would you add another note where the tonnage received is null but reused is not null?

Reply:
Good idea, have added